src/js/components/Chat.jsx

```diff
@@ -68,16 +66,16 @@ class ConnectedChat extends Component {
     }
 
     renderMessage = m => {
-        const {mid, created_on, message, uid, likes, dislikes, image, img} = m;
+        const {mid, created_on, message, uid, likes, dislikes, image, uploaded_image} = m;
```

Comment:
You cannot reuse 
```python 
image 
``` 
?

Reply:
<img width="980" alt="Screen Shot 2019-04-05 at 9 07 08 PM" src="https://user-images.githubusercontent.com/16313092/55662967-d9d1a300-57e6-11e9-9662-bc9d6fc94857.png">

Reply:
If I use image para los dos, se rompen los incoming messages. No tiene forma de saber si es una foto que viene del browser o es una foto que viene de un URL

Reply:
```javascript
image = typeof image.name == 'string' ? URL.createObjectURL(image): image;
```
or
```javascript
image = image instanceof File ? URL.createObjectURL(image): image;
```